AkochB1Onboarding/app/src/main/res/navigation/nav_graph.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<navigation xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto" android:id="@+id/nav_graph"
+    app:startDestination="@id/blockListFragment">
+
+    <fragment
+        android:id="@+id/blockListFragment"
+        android:name="com.example.akochb1onboarding.activity.BlockListFragment"
+        android:label="BlockListFragment" >
+        <action
+            android:id="@+id/action_blockListFragment_to_blockDetailFragment"
+            app:destination="@id/blockDetailFragment"
+            app:enterAnim="@anim/slide_in_right"
+            app:exitAnim="@anim/slide_out_left"
+            app:popEnterAnim="@anim/slide_in_left"
+            app:popExitAnim="@anim/slide_out_right" />
+    </fragment>
+    <fragment
+        android:id="@+id/blockDetailFragment"
+        android:name="com.example.akochb1onboarding.activity.BlockDetailFragment"
+        android:label="BlockDetailFragment" >
+        <argument
+            android:name="blockInfo"
+            app:argType="com.example.akochb1onboarding.domain.entity.Block" />
```

Comment:
I read in codelabs recommendations that is better always set a default value in these arguments and handle it with a switch from ktx